x-pack/packages/security-solution/upselling/pages/translations.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const UPGRADE_LICENSE_MESSAGE = (requiredLicense: string) =>
+  i18n.translate('securitySolutionPackages.entityAnalytics.paywall.upgradeLicenseMessage', {
+    defaultMessage: 'This feature is available with {requiredLicense} or higher subscription',
+    values: {
+      requiredLicense,
+    },
+  });
+
+export const UPGRADE_PRODUCT_MESSAGE = (requiredProduct: string) =>
+  i18n.translate('securitySolutionPackages.entityAnalytics.paywall.upgradeProductMessage', {
+    defaultMessage: 'Entity Analytics is available with {requiredProduct} subscriptions',
```

Comment:
We should not use the word "subscription" in serverless
```suggestion
    defaultMessage: 'Update your project features with {requiredProduct}',
```

Reply:
It is aligned with the design on this ticket: https://github.com/elastic/security-team/issues/7310
@SourinPaul @craig-abbott Could you please consider this comment and provide a new copy?

I would suggest `Entity risk scoring is available with SIEM Complete tier` but I am unsure if the user would understand.